src/implementations/twig/components/checkbox/checkbox.story.js

```diff
@@ -14,35 +7,137 @@ import dataDefault from '@ecl/specs-component-checkbox/demo/data--default';
 import checkboxGroup from './checkbox-group.html.twig';
 import notes from './README.md';
 
-const prepareCheckbox = (data) => {
-  getFormGroupKnobs(data);
+const getArgTypes = (data) => {
+  const argTypes = {};
+  argTypes.label = {
+    name: 'legend',
+    type: { name: 'string', required: true },
+    defaultValue: data.label,
+    description: 'The heading of the checkbox group',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+    control: {
```

Comment:
I investigated a bit the possibilities of reducing the syntax we use for defining the argTypes and we basically don't need to specify the control, it is assumed from the type key of the argType you define, and string is fine enough for storybook, i haven't tried but i bet the same is also true for the booleans you have.

Reply:
I did this work of returning the form elements controls through a utility in https://github.com/ec-europa/europa-component-library/pull/1835 ;)